features/auth/signup/CommunityGuidelinesForm.tsx

```diff
@@ -32,7 +32,7 @@ export default function CommunityGuidelinesForm() {
 
   const errorMessage = isHttpError(mutation.error)
     ? mutation.error?.error_messages[0]
-    : "";
+    : t("global:error.fatal_message");
```

Comment:
Getting a compile/type error here. I think it's just `error.fatal_message` or you'd need to wrap the namespace passed to `useTranslation` with a list.